github_connector/models/github_issue.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2016-Today: Odoo Community Association (OCA)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+
+from openerp import models, fields, api
+
+_logger = logging.getLogger(__name__)
+
+try:
+    import markdown
+except ImportError:
+    _logger.debug("Cannot import 'markdown' python librairy.")
+
+
+class GithubIssue(models.Model):
+    _name = 'github.issue'
+    _inherit = ['abstract.github.model.author']
+
+    _github_type = 'issue'
+    _github_login_field = 'number'
+
+    # Column Section
+    repository_id = fields.Many2one(
+        comodel_name='github.repository', string='Repository', readonly=True,
+        required=True, ondelete='cascade')
+
+    issue_type = fields.Selection(
+        selection=[('issue', 'Issue'), ('pull_request', 'Pull Request')],
+        string='Type')
+
+    title = fields.Char(string='Title', readonly=True, required=True)
+
+    body = fields.Char(string='Body', readonly=True)
+
+    html_body = fields.Html(
+        string='HTML Body', readonly=True, compute='_compute_html_body',
+        store=True)
+
+    state = fields.Selection(selection=[
+        ('open', 'Open'), ('closed', 'Closed')],
+        string='State', readonly=True, required=True)
+
+    comment_ids = fields.One2many(
+        string='Comments', comodel_name='github.comment',
+        inverse_name='issue_id', readonly=True)
+
+    comment_qty = fields.Integer(
+        string='Comments Quantity', compute='_compute_comment_qty',
+        store=True)
+
+    approved_comment_qty = fields.Integer(
+        string='Approved Comments Quantity', compute='_compute_opinion',
+        multi='opinion', store=True)
+
+    disapproved_comment_qty = fields.Integer(
+        string='Disapproved Comments Quantity', compute='_compute_opinion',
+        multi='opinion', store=True)
+
+    # Compute section
+    @api.multi
+    @api.depends('body')
+    def _compute_html_body(self):
+        for issue in self:
+            if issue.body:
+                issue.html_body = markdown.markdown(issue.body)
```

Comment:
don't we need some exception handling here in case the markdown parsing fails?

(real question, I don't know that python module at all)